include/qpid/dispatch/container.h

```diff
@@ -68,6 +68,32 @@ typedef enum {
 } qd_detach_type_t;
 
 
+/**
+ * Session Class
+ *
+ * Used when creating new links from the router.  A connection maintains a set
+ * of sessions over which links can be created.  The session class indicates
```

Comment:
The session class is used to obtain the appropriate session from the array of sessions that are housed in the connection. The maximum number of sessions in an inter-router connection is QD_SSN_CLASS_COUNT.